tests/components/withings/test_config_flow.py

```diff
@@ -62,7 +62,8 @@ async def test_config_reauth_profile(
 
     result = await hass.config_entries.flow.async_init(
         const.DOMAIN,
-        context={"source": config_entries.SOURCE_REAUTH, "profile": "person0"},
+        context={"source": config_entries.SOURCE_REAUTH},
```

Comment:
It would be good to start the flow with the complete context as done in the code under test.

https://github.com/home-assistant/core/blob/0d6bf08ff69a0b75ffe797f49c201d1b7c4f3a72/homeassistant/config_entries.py#L644-L649